data_ocean/models.py

```diff
@@ -3,11 +3,20 @@
 from django.utils.translation import gettext_lazy as _
 
 
+class DataOceanManager(models.Manager):
+    # exclude soft-deleted objects from queryset
+    def get_queryset(self):
+        return super().get_queryset().exclude(deleted_at__isnull=False)
```

Comment:
NOT (deleted_at NOT IS NULL) ?
```suggestion
        return super().get_queryset().filter(deleted_at__isnull=True)
```
deleted_at IS NULL

Reply:
first query: cost=0.29..21854.25 rows=45605 width=1185
second guery: cost=0.29..21854.25 rows=45605 width=1185